### PR TITLE
use absolute class name in require

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -24,7 +24,7 @@
 #
 # $plugin_prefix::              String which is prepended to the plugin package names
 #
-# $use_vhost::                  Enclose apache configuration in <VirtualHost>...</VirtualHost>
+# $use_vhost::                  Enclose apache configuration in VirtualHost tags
```
Comment (mmoll) [inline manifests/init.pp]: trying to ninja this in, related to theforeman/theforeman.org#636

File: manifests/install.pp [hunk 2]
```diff
@@ -29,8 +29,8 @@
   if $::foreman::passenger and $::foreman::passenger_ruby_package {
     package { $::foreman::passenger_ruby_package:
       ensure  => installed,
-      require => Class['apache'],
-      before  => Class['apache::service'],
+      require => Class['::apache'],
```
Comment (domcleal) [inline manifests/install.pp]: I think these can be left out, if the lint check is fixed.

**Comment by domcleal** [comment]
:+1: to the first two changes.

File: manifests/install.pp (hunk 2 above)
Comment (domcleal) [inline manifests/install.pp]: It's fixed, this shouldn't be necessary.

**Comment by mmoll** [comment]
corrected.

**Comment by domcleal** [comment]
:+1: subject to tests.

**Comment by domcleal** [comment]
Thanks @mmoll!

**Pull request merged**